[2024-04-12T20:33:55.729+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:33:52.156171+00:00 [queued]>
[2024-04-12T20:33:55.745+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:33:52.156171+00:00 [queued]>
[2024-04-12T20:33:55.748+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-12T20:33:55.772+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 20:33:52.156171+00:00
[2024-04-12T20:33:55.790+0000] {standard_task_runner.py:60} INFO - Started process 2053 to run task
[2024-04-12T20:33:55.809+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T20:33:52.156171+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp_gvtpitz']
[2024-04-12T20:33:55.824+0000] {standard_task_runner.py:88} INFO - Job 61: Subtask insert_qdrant
[2024-04-12T20:33:55.970+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T20:33:52.156171+00:00 [running]> on host 5feba290cccc
[2024-04-12T20:33:56.284+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T20:33:52.156171+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T20:33:52.156171+00:00'
[2024-04-12T20:33:56.355+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-12T20:33:56.388+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T203352, start_date=20240412T203355, end_date=20240412T203356
[2024-04-12T20:33:56.442+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-12T20:33:56.463+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
